### DISCUSSION TITLE
Can't Find Server Configuration Examples for Compression

### DISCUSSION THREAD

**Discussion by MilanHarrisDev** [jeffreylanters/react-unity-webgl#324] [Question and Answer]
Hello,
I am trying to use brotli compressed files with this module. I have found the section describing that I must configure my server to decompress when the files are requested. Your documentation says "see Server Configuration Code Samples". I can't seem to find these samples. Could you provide a link? Sorry if I missed them

**Comment by jeffreylanters** [comment]
This is not related to this module. But if you're using NGINX for example, you'll have to search on the internet for "NGINX Brotli configuration".

**Reply by MilanHarrisDev** [reply]
That makes sense, thank you! I was under the impression you had code examples, sorry!

**Reply by jeffreylanters** [reply]
Ah sorry for the confusion, I'll update the documentation! I hope it helped you. Happy coding!

**Comment by angela318** [comment]
@MilanHarrisDev Did you able to get the compression to work? Wonder if there is any documentation that I can refer on where to start.  It is taking too long to load without compression
